engine/api/cache/redis.go

```diff
@@ -322,7 +322,7 @@ func (s *RedisStore) Subscribe(channel string) PubSub {
 // GetMessageFromSubscription from a redis PubSub
 func (s *RedisStore) GetMessageFromSubscription(c context.Context, pb PubSub) (string, error) {
 	if s.Client == nil {
-		log.Error("redis> cannot get redis client")
+		log.Error("%v", sdk.WithStack(fmt.Errorf("cannot get redis client")))
```

Comment:
i think this log appears bunch of times in test logs, won't it be to verbose with this ?

Reply:
Yes it is, used for debug purpose, removed.